src/seedsigner/views/seed_views.py

```diff
@@ -187,10 +178,12 @@ def run(self):
             self.SEED_QR,
             self.TYPE_12WORD,
             self.TYPE_24WORD,
-            self.CREATE,
         ]
+
         if self.settings.get_value(SettingsConstants.SETTING__ELECTRUM_SEEDS) == SettingsConstants.OPTION__ENABLED:
-            button_data.insert(len(button_data)-1, self.TYPE_ELECTRUM)
+            button_data.append(self.TYPE_ELECTRUM)
```

Comment:
Would just would seem more consistent to keep the "Enter" options together than separated by create but not a biggie either way